management-service/services/configuration_service.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/caraml-dev/xp/common/api/schema"
+	"github.com/caraml-dev/xp/management-service/config"
+)
+
+type ConfigurationService interface {
+	GetTreatmentServiceConfig() schema.TreatmentServiceConfig
+}
+
+type configurationService struct {
+	treatmentServiceConfig schema.TreatmentServiceConfig
+}
+
+func NewConfigurationService(cfg *config.Config) ConfigurationService {
+	var segmenterConfig schema.SegmenterConfig
+	segmenterConfig = cfg.SegmenterConfig
+
+	// Iterates through all Sentry config labels to cast them as the type interface{}
+	sentryConfigLabels := make(map[string]interface{})
```

Comment:
We don't need this anymore, right?

Reply:
Oops yes we don't need that anymore 😅 Thanks for catching this!